aesara/link/jax/jax_linker.py

```diff
@@ -59,7 +61,17 @@ def create_jax_thunks(self, compute_map, storage_map):
             n for n, i in enumerate(self.fgraph.inputs) if isinstance(i, Constant)
         ]
 
-        thunk_inputs = [storage_map[n] for n in self.fgraph.inputs]
+        thunk_inputs = []
+        for n in self.fgraph.inputs:
+            sinput = storage_map[n]
+            if isinstance(sinput[0], RandomState):
+                new_value = jax_typify(sinput[0], getattr(sinput[0], "dtype", None))
```

Comment:
As you can probably tell by now, I'm always hesitant to include very type-specific hard-coded conditions&mdash;especially for a type/class like `RandomState` that has no subclasses.

I'm wondering if `RandomState` should/could be `SharedVariable` instead.

Reply:
Yeah, that could be useful, from a future perspective. We'll probably be changing it to `SharedVariable` or something else once we have other arguments that need to be `jax_typified`.

Reply:
Let's try this change now, since it would be better to establish this general framework sooner than later.